UAT/V1.13.3.patch.sql

```diff
@@ -0,0 +1,40 @@
+-- https://trello.com/c/BmEiIfud
+
+DROP TYPE IF EXISTS cqc.owner_change_status;
```

Comment:
Please rename TYPE name to "OwnerChangeStatus" instead of owner_change_status and everywhere else too to keep all other objects in sync..

Reply:
Changed and committed.